dev/default.env

```diff
@@ -8,7 +8,7 @@
 # https://docs.docker.com/compose/reference/envvars/#compose_project_name
 # Containers started with the below value will have their names prefixed with it
 # Required on shared infrastructure
-# COMPOSE_PROJECT_NAME=dem-dev
+# COMPOSE_PROJECT_NAME=helloworld-dev
 
```

Comment:
I've found if this is left commented, people will leave it as-is, causing future problems
```suggestion
# Choose something unique to the docker host, eg ${INSTANCE_ID}-dev-${USER}
COMPOSE_PROJECT_NAME=

```